messages/web.login.md

```diff
@@ -49,3 +49,11 @@ If you don’t specify --browser, the command uses your default browser. The exa
 # invalidClientId
 
 Invalid client credentials. Verify the OAuth client secret and ID. %s
+
+# error.cannotOpenBrowser
+
+Unable to open %s.
```

Comment:
```suggestion
Unable to open the browser you specified (%s).
```

Reply:
FYI @cristiand391 , two small edits.  Okay?

Reply:
yep, thank you!